client/src/FindSchool.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Button, ButtonGroup } from "@chakra-ui/react";
```

Comment:
ButtonGroup isn't used so we don't need to import unnecessary components :) (it will slow the app down unnecesarily)

Reply:
Ok, I deleted it